src/App/App.tsx

```diff
@@ -37,7 +37,7 @@ const App = () => {
     const isNetworkVisualization = usePage('/netwerkvisualisatie')
 
     if (process.env.NODE_ENV !== 'production' && !process.env.JEST_WORKER_ID) {
-        axe()
+        // axe()
```

Comment:
@Aidenbuis Can you add a new environment variable so you don't have to comment this line? Then we can just deactivate this plugin using our .env file.

Reply:
Done in other branch:
https://github.com/Provincie-Zuid-Holland/Omgevingsbeleid-Frontend/pull/721/files#diff-1deef93890e359e3ea9cccd8b9fb992a1008c1f21b9eee105a1a364afae4f170